basic_flutter/lib/splash_screen.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+void main() {
```

Comment:
remove the main function, you don't need it there, just include the SplashScreen as a widget to be pushed